### PR TITLE
doc/md: turso guide

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: doc/md/guides/sqlite/turso.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+      unique  = true
+      columns = [column.name]
+   }
+}
+schema "main" {
+}
+```
+</details>
+You can also use the `--format` flag to change the output format. For example, you can use
+the `--format json` flag to print the schema in JSON format:
```
Comment (a8m) [inline doc/md/guides/sqlite/turso.md:201]:
```suggestion
the `--format '{{ sql . "  " }}'` flag to print the schema in JSON format:
```
I'd go with SQL as it's more useful.

File: doc/md/guides/sqlite/turso.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
+---
+id: turso
+title: Working with Turso
+slug: /guides/sqlite/turso
+---
+[Turso](https://turso.tech) is an edge-hosted, distributed database based on libSQL, an open-source
```
Comment (hilakashai) [inline doc/md/guides/sqlite/turso.md:6]: link to libSQL

File: doc/md/guides/sqlite/turso.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+and open-contribution fork of SQLite. It was designed to minimize query latency
+for applications where queries come from anywhere in the world. 
+
+Engineers can use Atlas to manage their Turso databases using the SQLite driver by providing
```
Comment (hilakashai) [inline doc/md/guides/sqlite/turso.md:10]:
```suggestion
Engineers can use Atlas to manage their Turso databases using the SQLite driver by
```

**Review by a8m — APPROVED** [review]

**Review by hilakashai — COMMENTED** [review]
💯

**Pull request merged**